language/cloud-client/v1/snippets.py

```diff
@@ -240,12 +279,20 @@ def entity_sentiment_file(gcs_uri):
             print(u'  Type : {}'.format(mention.type))
         print(u'Salience: {}'.format(entity.salience))
         print(u'Sentiment: {}\n'.format(entity.sentiment))
-# [END language_entity_sentiment_gcs]
+    # [END language_entity_sentiment_gcs]
 
 
-# [START language_classify_text]
-def classify_text(text):
-    """Classifies content categories of the provided text."""
+def classify_text():
+    # [START language_classify_text]
+    import six
+    from google.cloud import language
+    from google.cloud.language import enums
+    from google.cloud.language import types
+
+    text = 'Android is a mobile operating system developed by Google, ' + \
```

Comment:
The `+` signs are not necessary here. The string will continue because of the `\`.

Reply:
Very interesting. I did not know this little trick. Thanks.  :)